website/docs/components/tooltip/partials/guidelines/guidelines.md

```diff
@@ -2,34 +2,81 @@
 
 ### When to use
 
-- {description}
+When displaying additional context or information that is not critical to the user's task or decision-making process.
 
 ### When not to use
 
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
-- {description}, consider {[component](#)}.
+- When the content is too lengthy or complex.
+- When the information provided is critical to the user's task or decision-making process, consider moving the content to the page or an [alert](https://helios.hashicorp.design/components/alert).
 
-### Variant/property name
+tooltip-when-to-use
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+!!! Do
 
-### Variant/property name
+![Correct use of a tooltip](/assets/components/tooltip/tooltip-when-to-use.png)
 
-<!-- don’t forget to include real examples and do/don’t blocks, as necessary -->
-{description}
+!!!
+
+!!! Dont
+
+![Incorrect use of a tooltip](/assets/components/tooltip/tooltip-when-not-to-use.png)
+
+!!!
+
+
+## Placement
+
+Tooltips can be placed in various positions relative to the UI element they are associated with. Choose the placement based on the context and available space around the tooltip trigger.
+
+There are twelve placement options: `top`, `bottom`, `left`, `right`, `top-start`, `top-end`, `bottom-start`, `bottom-end`, `left-start`, `left-end`, `right-start`, `right-end`.
+
```

Comment:
I think it's confusing to say there are twelve placement options but then show a picture of only four.

I would change the text to:

"There are four main placement options: `top`, `bottom`, `left`, and `right`."

You _could_ mention that there are other placement options as well but, personally, I don't think the other options are really useful or necessary and I think listing them all is likely to be more confusing than helpful.